reconcile/external_resources/aws.py

```diff
@@ -25,6 +26,11 @@ def resolve(self, spec: ExternalResourceSpec) -> dict[str, Any]: ...
     @abstractmethod
     def validate(self, resource: ExternalResource) -> None: ...
 
+    def find_linked_resources(
+        self, spec: ExternalResourceSpec
+    ) -> set[ExternalResourceKey]:
+        return set()
```

Comment:
Please add a doc string here, mainly because it's the base class.

Reply:
Added.